src/main/kotlin/no/nav/familie/ba/sak/ekstern/restDomene/RestPersonResultat.kt

```diff
@@ -78,3 +78,15 @@ fun PersonResultat.tilRestPersonResultat() =
                 annenVurdering.tilRestAnnenVurdering()
             },
     )
+
+fun RestPersonResultat.fjernAutomatiskBegrunnelse() =
+    copy(
+        vilkårResultater =
+            vilkårResultater.map { vilkårResultat ->
+                if (vilkårResultat.begrunnelse == "Fylt inn automatisk fra registerdata i PDL") {
+                    vilkårResultat.copy(begrunnelse = "")
```

Comment:
```suggestion
                    vilkårResultat.copy(begrunnelse = "", erAutomatiskVurdert = false)
```
Bør denne settes til false?

Reply:
Usikker, ser den blir satt til true for UNDER_18 og GIFT_PARTNERSKAP når vi preutfyller de så tenkte det gav mening å sette den til true. Så ut som det hovedsakelig ble brukt til tekst i begrunnelse-feltet i frontend.

Reply:
Ja, jeg er enig i at den bør settes til true initielt, men bør den endres til false når saksbehandler gjør endringer på vilkåret? Har ikke noen sterke meninger om det, og hvis den bare brukes i frontend for visuelle ting har det kanskje ikke all verdens å si